### ISSUE TITLE
Dependencies issue

### ISSUE THREAD

**Issue by joaquin-exe** [GlitchedPolygons/cecies#5]
Hi, I was trying to use your code and when using encrypt.c I got the error that ccrush.h could not be found. However when I search the internet I can not find the ccrush library anywhere. Thank you in advance for your help.

I am pretty new to coding so my issue might be due to some beginner mistake.

**Comment by GlitchedPolygons** [comment]
Hi there,

So cool that you found my library :D

And welcome to coding! I wish you lots of success and patience with this fascinating activity in life. If you never stop being curious and thirsty for learning new things you'll be fine and love coding as much as I do :)

ccrush is a library (actually more of a Zlib wrapper) that I created for compressing and decompressing data in C projects with a single function call.

https://github.com/GlitchedPolygons/ccrush

It was added to CECIES as a git submodule: perhaps you forgot to do a recursive clone and thus your project doesn't have it at all in that folder? Or maybe you forgot to add its `include/` directory to your project's header inclusion path? (When using CMake, this is what `target_include_directories()` does).

Cheers, and all the best!

**Comment by joaquin-exe** [comment]
Ok, I see. I am not very comfortable with github yet, that's why I didn't clone your library and I simply downloaded the binaries from here: https://github.com/GlitchedPolygons/cecies/releases. I am running ubuntu 18.04 so I downloaded this tar file only: cecies-4.0.0-linux-x86_64.tar.gz
I guess that didn't include the ccrush library and that is the problem.

Anyways, I was using your library to implement the ECIES scheme used in 5G networks, but apparently this schem uses the ANSI x9.63 kdf which is not included in the mbedtls library I think: https://github.com/ARMmbed/mbedtls/issues/2335, so I might have to use another library I guess
